modules/apps/frontend-js/frontend-js-state-web/src/main/resources/META-INF/resources/State.ts

```diff
@@ -58,7 +58,7 @@ const warnings = {
 	writeKey: new Set(),
 };
 
-const State = {
+export const State = {
```

Comment:
We previously were exporting State as `default`, so we probably shouldn't move it to a named export instead

Reply:
True, that was a bad change prompted by me not being able to get the red squiggly lines away from VSC, but they're gone now.